src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,49 +1,60 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
 import React from 'react';
 import { Loader } from '../Loader';
+import { UserInfo } from '../../types/UserInfo';
 
-export const TodoModal: React.FC = () => {
-  return (
-    <div className="modal is-active" data-cy="modal">
-      <div className="modal-background" />
-
-      {true ? (
-        <Loader />
-      ) : (
-        <div className="modal-card">
-          <header className="modal-card-head">
-            <div
-              className="modal-card-title has-text-weight-medium"
-              data-cy="modal-header"
-            >
-              Todo #2
-            </div>
-
-            {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-            <button
-              type="button"
-              className="delete"
-              data-cy="modal-close"
-            />
-          </header>
-
-          <div className="modal-card-body">
-            <p className="block" data-cy="modal-title">
-              quis ut nam facilis et officia qui
-            </p>
-
-            <p className="block" data-cy="modal-user">
-              {/* <strong className="has-text-success">Done</strong> */}
-              <strong className="has-text-danger">Planned</strong>
+interface Props {
+  userInfo: UserInfo | null;
+  loaderStatus: boolean;
+  closeModal: () => void;
+}
 
-              {' by '}
+export const TodoModal: React.FC<Props> = ({
+  userInfo,
+  loaderStatus,
+  closeModal,
+}) => (
+  <div className="modal is-active" data-cy="modal">
+    <div className="modal-background" />
 
-              <a href="mailto:Sincere@april.biz">
-                Leanne Graham
-              </a>
-            </p>
+    {loaderStatus ? (
+      <Loader />
+    ) : (
+      <div className="modal-card">
+        <header className="modal-card-head">
+          <div
+            className="modal-card-title has-text-weight-medium"
+            data-cy="modal-header"
+          >
+            {`Todo #${userInfo?.todo.id}`}
           </div>
+
+          <button
+            type="button"
+            className="delete"
+            data-cy="modal-close"
+            onClick={closeModal}
+          />
+        </header>
+
+        <div className="modal-card-body">
+          <p className="block" data-cy="modal-title">
+            {userInfo?.todo.title}
+          </p>
+
+          <p className="block" data-cy="modal-user">
+            {userInfo?.todo.completed ? (
+              <strong className="has-text-success">Done</strong>
+            ) : (
+              <strong className="has-text-danger">Planned</strong>
+            )}
+
+            {' by '}
```

Comment:
adding spaces this way is a bad practice, you can use margins and paddings for that